app/routes.js

```diff
@@ -93,6 +93,9 @@ module.exports = function(express,envVariables) {
             }
         }
 
+        if (req.query.next && req.query.next === 'continueEApp') {
+            req.session.continueEAppFlow = true;
+        }
```

Comment:
This is needed to continue the eApp flow if someone has started the process before signing in.